source/index.ts

```diff
@@ -360,6 +360,11 @@ is.nonEmptySet = <T = unknown>(value: unknown): value is Set<T> => is.set(value)
 is.emptyMap = (value: unknown): value is Map<never, never> => is.map(value) && value.size === 0;
 is.nonEmptyMap = <Key = unknown, Value = unknown>(value: unknown): value is Map<Key, Value> => is.map(value) && value.size > 0;
 
+/**
+Returns `true` if `value` can be used as an object property key (either a `string`, a `number` or a `symbol`).
```

Comment:
The text here should be exactly the same as in the readme. Notice the `a`'s.

Reply:
@sindresorhus why is a comment needed in the cod? I don't see similar cases of comments in the code.